mlx/backend/common/conv.cpp

```diff
@@ -3,8 +3,12 @@
 #include <cassert>
 
 #ifdef ACCELERATE_NEW_LAPACK
+#if __has_include(<vecLib/vecLib.h>)
 #include <vecLib/cblas_new.h>
 #else
+#include <Accelerate/Accelerate.h>
+#endif
+#else
```

Comment:
on iOS you get this via the Accelerate umbrella header

Reply:
Does `Accelerate.h` provide all the headers even on MacOS? If so, we could just use that instead of `vecLib`...?

Reply:
It looks like it does actually:

```
#ifndef __VECLIB__
#include "../Frameworks/vecLib.framework/Headers/vecLib.h"
#endif
```

I will test that out.

Reply:
Tested it out and it works on both iOS and macOS -- updated the PR.